Composer/packages/client/src/components/QnA/utilities.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export const QnALanguageToLocale = (language: string): string => {
+  switch (language) {
+    case 'Chinese_Simplified':
+      return 'zh-cn';
+    case 'English':
+      return 'en-us';
+    case 'French':
+      return 'fr-fr';
+  }
+  return language;
```

Comment:
QnAMaker portal API returns `language` of KB, but we need` locale` to do filter. Need to find a better way than hard code

Reply:
Aah. This is a bit tricky. We do have have a locale file here https://github.com/microsoft/BotFramework-Composer/blob/main/Composer/packages/client/src/locales.ts and its in memory for us to parse back the locale from the language.

Reply:
I found the language/locale mapping of bf-lu publishing, 

https://github.com/microsoft/botframework-cli/blob/132e0cd1f5c393edc35fdc57f971b6cb8da94445/packages/lu/src/parser/utils/enums/localeToQnALanguageMap.js

And use it created a inverted map

![image](https://user-images.githubusercontent.com/49866537/123595114-16a06f00-d823-11eb-98b9-2e63406a1290.png)

finally list out all related locale KBs. @srinaath @boydc2014

Reply:
Good find @zhixzhan. Looks like bf-cli faced the same issue and built this file :)